client/scss/components/_main-nav.scss

```diff
@@ -120,6 +119,10 @@
     .account {
         display: none;
     }
+
+    *:focus {
+        @include show-focus-outline-inside;
+    }
```

Comment:
This improves the appearance of the outline in the main menu, which relies a lot on hidden overflows.